pridepy/download/client.py

```
@@ -0,0 +1,308 @@
+        """Retrieve project files belonging to the given categories."""
+        return registry.resolve(accession).get_category_files(accession, categories)
+
+    def get_submitted_file_path_prefix(self, accession):
+        """Shim — see :meth:`PrideProvider.get_submitted_file_path_prefix`."""
+        return PrideProvider().get_submitted_file_path_prefix(accession)
+
+    def get_file_from_api(self, accession, file_name) -> List[Dict]:
+        """Return records matching ``file_name`` from the provider's listing."""
+        try:
+            return registry.resolve(accession).find_file(accession, file_name)
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)